presets/4.3/tune/karate_freestle.txt

```diff
@@ -0,0 +1,87 @@
+Karate FS tune
+#$ TITLE: Karate Freestyle
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY 
+#$ KEYWORDS: karate, 6S,5 inch, 5", sugarK, limon, ctzsnooze, karatreBrot, freestyle
+#$ AUTHOR: sugarK
+#$ DESCRIPTION: This is the sugarK freestyle tune, its been test over the last few months by members of the UAVtech discord.. This is a starting point tune and has siders active so you can fine tune it easily.
+#$ DESCRIPTION: 
+#$ DESCRIPTION: Things to note.. YOU HAVE TO USE RPM FILTERING WITH THIS TUNE, failure to do so might result in fire and regarding Dshot600,  if your setup has errors in the motor tab using bidirectional Dshot then change to 8k/4k and dshot300.
+#$ DESCRIPTION: Also this should be applied to a clean flash after you've setup your rates, swtiches, vtx tables etc. To test arm the quad with props on, it should sound clean with no grinding, if it passes that then hover test and check motor temps.
+#$ DESCRIPTION:  If any thing is off don't fly it!!
+#$ DESCRIPTION: 
+#$ DESCRIPTION: SECOND NOTE.... Radio links.. 1. Make sure your radio system is totally up to date using either Edgetx or Opentx and your ADC in the hardware page is OFF 2. Go to the radio presets and apply the correct setup for your system and link speed.
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/95
+#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
+#$ INCLUDE: presets/4.3/tune/defaults.txt
+#$ INCLUDE: presets/4.3/filters/defaults.txt
+
+
+# -- Gyro lowpass filters --
+set gyro_lpf2_static_hz = 1000
+set gyro_lpf1_type = PT1
+set gyro_lpf2_type = PT1
+set gyro_lpf1_dyn_min_hz = 500
+set gyro_lpf1_dyn_max_hz = 1000
+set simplified_gyro_filter_multiplier = 200
+
+# -- Gyro Dynamic Notches --
+set dyn_notch_count = 2
+set dyn_notch_q = 500
+set dyn_notch_min_hz = 200
+
+# -- Dterm filtering --
+set dterm_lpf1_dyn_expo = 10
+
+# -- RPM filtering --
+set dshot_bidir = ON
+set rpm_filter_fade_range_hz = 100
+
+# -- Misc --
+set yaw_spin_recovery = AUTO
+set thrust_linear = 0
```

Comment:
thrust_linear  = 0 and yaw_spin_recovery = AUTO are like that in TUNE defaults anyways.

Reply:
good point